lib/usd/ui/layerEditor/layerEditorWidget.cpp

```diff
@@ -54,8 +54,16 @@ using namespace UsdLayerEditor;
 // properly support high dpi with style sheets
 QString getDPIPixmapName(QString baseName)
 {
-    QString suffix(QString::number(IS_MAC_OS ? 100 : DPIScale(100)));
-    return baseName + "_" + suffix + ".png";
+#ifdef Q_OS_DARWIN
+    return baseName + "_100.png";
+#else
+    const auto scale = utils->dpiScale();
+    if (scale >= 2.0)
+        return baseName + "_200.png";
+    else if (scale >= 1.5)
+        return baseName + "_150.png";
+    return baseName + "_100.png";
+#endif
```

Comment:
When running at custom scaling 125%, the base name would become "<basename>_125.png" - but we do not provide at 125% icon file. So reworked the code to make sure we always return either _100/_150/_100 - which are the three sizes we support.